adops/setting-up-prebid-native-in-dfp.md

```diff
@@ -21,34 +21,33 @@ For more information about DFP native ad setup, see the [DFP native ads document
 * TOC
 {:toc}
 
-## 1. Add a native ad unit
+## 1. Create a native ad unit
 
-You can specify a size for the ad unit, or specify the "fluid" size.  In this case we'll go with **Fluid**.
+Under **Display > Native**, select the **HTML & CSS editor** option.
```

Comment:
I'm not seeing these options.  Can we include a screenshot here?

Reply:
I think you're talking about creating a new native ad here, not a new "ad unit", but this is a bit confusing.

Reply:
clarified and added screen shots